docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+        defaultValue: {
+          summary: 'stretch',
+        }
+      }
+    },
+    narrow_align: {
+      options: ['inherit', 'stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Align wrap
+
+    alignWrap: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets how stack lines are distributed, if they `wrap` into multiple lines. This has equivalent behavior to the `align-content` Flexbox property.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        }
+      }
+    },
+
+    narrow_alignWrap: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Spread
+
+    spread: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'Sets how items will be distributed in the stacking direction.', 
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        },
+      },
+    },
+
+    narrow_spread: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Wrap
+
+    wrap: {
+      options: ['wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets whether items are forced onto one line or can wrap onto multiple lines.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'nowrap',
+        }
+      }
+    },
+    narrow_wrap: {
+      options: ['inherit', 'wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Divider
+
+    showDividers: {
+      control: {
+        type: 'boolean'
+      },
+      description: `Whether a divider between items is shown or not.
+
+_Note: the presence of a divider duplicates the \`gap\` between items._`,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'false',
+        }
+      }
+    },
+    dividerAriaRole: {
+      options: ['presentation', 'separator', 'none'],
```

Comment:
"presentation" and "none" do the same thing. If we really need the option to add `role="separator"`, maybe we should reconsider the API for this prop.